bug/bug.go

```diff
@@ -667,8 +667,10 @@ func (bug *Bug) LastOp() Operation {
 // Compile a bug in a easily usable snapshot
 func (bug *Bug) Compile() Snapshot {
 	snap := Snapshot{
-		id:     bug.id,
-		Status: OpenStatus,
+		id:           bug.id,
+		Actors:       bug.computeActors(),
```

Comment:
You think it's better to have one function that compute both actors and participants in order to loop only once on operations ?